dockerfiles/init/manifests/codenvy.env

```diff
@@ -473,3 +473,14 @@
 #     an expanded list of nodes. You should not have to configure this as we
 #     manage it with the CLI.
 #CODENVY_SWARM_NODES=172.17.0.1:2375
+
+
+############################################################
+#####                                                  #####
+#####                CODENVY EXTRA HOSTS               #####
+#####                                                  #####
+# Extra hosts
+#     A comma separated list of <HOST>:<IP> combinations which will be added to /etc/hosts file of
+#     each codenvy container. This could be useful for same cases where codenvy system should be aware
```

Comment:
each codenvy container here can lead to user's believing that it will be used in workspaces too.